ledger/participant-integration-api/src/main/scala/platform/apiserver/error/LedgerApiErrors.scala

```diff
@@ -61,6 +61,26 @@ object LedgerApiErrors extends LedgerApiErrorGroup {
   }
 
   object CommandValidation extends ErrorGroup {
+
+    @Explanation(
+      """Event id missing PBATKO TODO"""
+    )
+    @Resolution("Ensure that your application is correctly configured to send valid event ids")
+    object EventIdMismatch
+        extends ErrorCode(
+          id = "EVENT_ID_MISMATCH",
+          ErrorCategory.InvalidGivenCurrentSystemStateResourceMissing,
+        ) {
+
+      case class Reject(override val cause: String)(implicit
```

Comment:
Created new error because I didn't find any of the existing ones fit the V1 error from ApiTransactionService.scala that I used as a testing ground.

We can delete it and limit this PR to consist only of an unused (aside from its unit tests) `ErrorCodesVersionsSwitcher`